### DISCUSSION TITLE
Seção 3: Terraform básico

### DISCUSSION THREAD

**Discussion by guilhermesaid** [chgasparoto/curso-aws-com-terraform#76] [Q&A]
Olá @chgasparoto .

Nos códigos que está ensinando até onde vi você esta criando as instancias sem nenhum par de chaves para a conexão, neste caso qual string eu poderia adicionar ao código para adicionar um par de chave já criado no ec2?

Abraço!

**Comment by chgasparoto** [comment]
Fala @guilhermesaid , você tem que usar essa propriedade https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/instance#key_name

Abraço.
